scripts/sync.sh

```diff
@@ -44,7 +44,11 @@ function sync {
     # /bin/bash -c execute commands in " " in the new environment 
     # envsubst: replace all env variables found ($f) with the ones specified 2 steps before, pipe to sponge and write to file
     for f in $(find "$workdir/.github/ISSUE_TEMPLATE" "$workdir/.github/pull_request_template.md" "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md" "$workdir/CODE_OF_CONDUCT.md" "$workdir/LICENSE" -type f -print); do
-        env -i REPOSITORY="$project" PROJECT="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
+        if [[ "$project" = "hydra" || $project = "kratos" || $project = "oathkeeper" || $project = "keto" ]]; then
+        env -i discussions="$discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
+        else
+        env -i discussions="https://github.com/meta/discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```

Comment:
Please format and indent.

```suggestion
            env -i discussions="https://github.com/meta/discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```